internal/storage/db/sqlite3/sqlite3.go

```diff
@@ -20,12 +21,13 @@ import (
 	"go.uber.org/zap"
 
 	// import sqlite3 driver.
```

Comment:
I think we can remove this comment and let the import be grouped with the rest. The comment was only there because the linter requires comments for blank imports.